paddle/gserver/layers/CudnnConvLayer.cpp

```diff
@@ -186,6 +164,12 @@ void CudnnConvLayer::forward(PassType passType) {
   reshape(batchSize);
   resetOutput(batchSize, outputH_ * outputW_ * numFilters_);
 
+  void* workSpace = NULL;
+  if (workSpaceInBytes_ > 0) {
+    MatrixPtr tmpMat = Matrix::getTmpMatrix(1, workSpaceInBytes_, true);
```

Comment:
Use `getTmpMatrix` is not very suitable. 
There need a device memory only used by all ConvLayers, so a global device memory with mutex is more suitable.